jctools-core/src/test/java/org/jctools/queues/QueueSanityTestMpscBlockingConsumerArrayExtended.java

```diff
@@ -375,11 +500,17 @@ public void testSpinWaitForUnblockForever() throws InterruptedException {
             }
 
             public void run() {
+                ArrayDeque<T> ints = new ArrayDeque<>();
                 try {
                     for (int i = 0; i < interations; ++i) {
                         T t;
                         do {
-                            t = source.poll(1,  TimeUnit.NANOSECONDS);
+                            if (useDrain) {
```

Comment:
this test should be in the extended MPQ test IMO

Reply:
I've moved it into the MPQ test but it's a shame will run twice because the queues needed by the test are allocated during the test itself and not upfront as the other MPQ tests